_posts/2022-09-21-hvr-and-hva.markdown

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+font: serif
+title: Why you should be using HVR for high volume data replication
+date: 2022-08-08
+highlight: monokai
+author: Matt Carter, Jack Menzie, Leon Ticharwa
+image: /img/blog/hvr-and-hva/banner.jpg
+tags: hvr hva data replication
+---
+
+## Why HVA/HVR ?
```

Comment:
We probably need some introduction before this. Maybe even talking about our training in Sydney and posting some photos of it

Reply:
@zmaleki If you've got any, post them in here. As far as I'm aware, we haven't got many suitable for the blog?